.gitlab/variables.yml

```diff
@@ -15,6 +15,7 @@
     BUILD_MPI: "OFF"
     MPI_AS_ROOT: "OFF"
     FAST_TESTS: "OFF"
+    NONDEFAULT_STREAM: "OFF"
```

Comment:
Do we have at least one test with the stream check on?

Reply:
yes, `build/cuda110/mvapich2/gcc/cuda/debug/shared` has a subsequent `test` job